web/index.html

```diff
@@ -17,237 +17,91 @@
         );
-
-        if (loc.lat === cachedLoc.lat && loc.lng === cachedLoc.lng) return;
-        cachedLoc = loc;
-        console.log("Move Marker: Trainer - " + loc.lat + ", " + loc.lng);
-        marker.setPosition({lat: parseFloat(loc.lat), lng: parseFloat(loc.lng)});
-        loc.cells.forEach(function(item, i, arr) {
-          item.forts.forEach(function(fort, i, arr) {
-            if(!(fort.id in markers)) {
-              markers[fort.id] = new google.maps.Marker({
-                map: map,
-                position: {lat: parseFloat(fort.latitude), lng: parseFloat(fort.longitude)}
-                //icon: "image/trainer-icon.png"
-              });
-              if(('type' in fort) && (fort.type == 1)){
-                markers[fort.id].setIcon("image/forts/Pstop.png")
-              } else {
-                if(fort.owned_by_team == 0)
-                  markers[fort.id].setIcon("image/forts/Gym.png")
-                if(fort.owned_by_team == 1)
-                  markers[fort.id].setIcon("image/forts/Mystic.png")
-                if(fort.owned_by_team == 2)
-                  markers[fort.id].setIcon("image/forts/Valor.png")
-                if(fort.owned_by_team == 3)
-                  markers[fort.id].setIcon("image/forts/Instinct.png")
-              }
-            }
-
-          });
-
-        });
+        
+        if (tLoc.lat === cachedtLoc.lat && tLoc.lng === cachedtLoc.lng) return;
+        cachedtLoc = tLoc;
+        console.log("Move Marker: Trainer - " + tLoc.lat + ", " + tLoc.lng);
+        tMarker.setPosition({lat: parseFloat(tLoc.lat), lng: parseFloat(tLoc.lng)});
+      }
+      function addCatchable() {
+        loadJSON('catchable.json',
+          function(data) { cInfo = data; },
+          function(xhr) { console.error(xhr); }
+        );
+        loadJSON('pokemondata.json',
+          function(data) { pokemonArray = data; },
+          function(xhr) { console.error(xhr); }
+        );
+        if(cInfo['latitude'] != undefined) {
+          if(cMarker == undefined) {
+            console.log("New Marker: Catchable Pokemon - " + parseFloat(cInfo['latitude']) + ", " + parseFloat(cInfo['longitude']));
+            cMarker = new google.maps.Marker({
+              map: map,
+              position: {lat: parseFloat(cInfo['latitude']), lng: parseFloat(cInfo['longitude'])},
+              icon: "image/icons/" + cInfo['pokemon_id'] + ".png"
+            });
+          }
+          console.log("Update Marker: Catchable Pokemon - " + parseFloat(cInfo['latitude']) + ", " + parseFloat(cInfo['longitude']));
+          cMarker.setPosition({lat: parseFloat(cInfo['latitude']), lng: parseFloat(cInfo['longitude'])});
+          cMarker.setIcon("image/icons/" + cInfo['pokemon_id'] + ".png");
+        }
+        if(cInfo['latitude'] == undefined && cMarker != undefined) {
+          cMarker.setIcon("image/icons/blank.png");
+        }
       }
       function updateInventory() {
         loadJSON('info.json',
```

Comment:
You need this one to get rid of browser cache:
`
loadJSON('info.json?_=' + new Date().getTime(),
`